auto_py_to_exe/web/js/i18n.js

```diff
@@ -1356,6 +1423,10 @@ const supportedLanguages = [
         name: 'Spanish Latam (Español Latam)',
         code: 'sp_la',
     },
+    {
+        name: 'Srpski (Serbian)',

```

Comment:
Could you swap these around (like the others)

Reply:
i've swapped it!